admin/class-automation.php

```diff
@@ -179,11 +207,11 @@ public static function filter__apple_news_exporter_slug( $slug, $post_id ) {
 	 * @return array Filtered JSON for the article.
 	 */
 	public static function filter__apple_news_generate_json( $json, $post_id ) {
-		$title = $json['title'] ?? '';
-		$rules = self::get_automation_for_post( $post_id );
+		$rules         = self::get_automation_for_post( $post_id );
+		$json['title'] = self::$original_title;
```

Comment:
Is there any reason to suspect that `filter__apple_news_exporter_title()` and this method might not run in the "correct" order, such that this static `$original_title` variable references the wrong post?

Since the post ID is available in both places, would it be overkill to make `$original_title` a map of ID > title that could be referenced here?

Reply:
They should never run in the wrong order. The exporter title is set up when the exporter object is set up, which happens before any of the logic to build the exported object. However, I like your suggestion about establishing the ID/title map, and will apply that.